engine/src/main/java/io/zeebe/engine/state/ZbColumnFamilies.java

```diff
@@ -42,6 +42,7 @@
 
   // jobs
   JOBS,
+  JOBS_CREATION,
```

Comment:
Weren't you the one who asked not to create new column families? :stuck_out_tongue_winking_eye:

Reply:
Jep but we didn't decided against it so :man_shrugging: